iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -455,3 +476,205 @@ def stop_translations(self):
+        cur = self.db_query(sql)
+        self.create_poll_question(poll, poll_row, cur)
+        cur.close()
+
+    def create_poll_question(self, poll, poll_row, cur):
+        PollFormField.objects.filter(page=poll).delete()
+        choices = []
+        for row in cur:
+            choices.append(row['title'])
+
+        choices_length = len(choices)
+
+        if choices_length == 2:
+            field_type = 'radio'
+        elif choices_length > 2:
+            if poll_row['allow_multiple_choice']:
+                field_type = 'checkboxes'
+            else:
+                field_type = 'dropdown'
+        else:
+            return
+
+        choices = ','.join(choices)
+
+        PollFormField.objects.create(page=poll, label=poll.title, field_type=field_type, choices=choices)
+        self.stdout.write(f"saved poll question, label={poll.title}")
+
+    def migrate_surveys(self, survey_index_page):
+        sql = "select * " \
+              "from surveys_molosurveypage smsp, wagtailcore_page wcp, core_languagerelation clr, core_sitelanguage csl " \
+              "where smsp.page_ptr_id = wcp.id " \
+              "and wcp.id = clr.page_id " \
+              "and clr.language_id = csl.id "
+        if self.skip_locales:
+            sql += "and locale = 'en' "
+        sql += 'order by wcp.path'
+        cur = self.db_query(sql)
+        survey_page_translations = []
+        for row in cur:
+            if row['page_ptr_id'] in self.page_translation_map:
+                survey_page_translations.append(row)
+            else:
+                self.create_survey(survey_index_page, row)
+        else:
+            for row in survey_page_translations:
+                survey = self.v1_to_v2_page_map.get(self.page_translation_map[row['page_ptr_id']])
+                locale, __ = Locale.objects.get_or_create(language_code=row['locale'])
+
+                self.translate_page(locale=locale, page=survey)
+
+                translated_survey = survey.get_translation_or_none(locale)
+                if translated_survey:
+                    translated_survey.title = row['title']
+                    translated_survey.draft_title = row['draft_title']
+                    translated_survey.live = row['live']
+                    translated_survey.description = self.map_survey_description(row['description'])
+                    translated_survey.thank_you_text = self.map_survey_thank_you_text(row)
+                    translated_survey.allow_anonymous_submissions = row['allow_anonymous_submissions']
+                    translated_survey.allow_multiple_submissions = row['allow_multiple_submissions_per_user']
+                    translated_survey.submit_button_text = row['submit_text'] or 'Submit'
+                    translated_survey.direct_display = row['display_survey_directly']
+                    translated_survey.multi_step = row['multi_step']
+                    translated_survey.save(
```

Comment:
Maybe not add update fields? Somebody doing this later might miss it

Reply:
done